pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -528,6 +553,21 @@ func (o *MustGatherOptions) copyFilesFromPod(pod *corev1.Pod) error {
 	if err := os.MkdirAll(destDir, os.ModePerm); err != nil {
 		return err
 	}
+
+	// get must-gather gather container logs
+	logOptions := &corev1.PodLogOptions{
+		Container:  gatherContainerName,
+		Follow:     false,
+		Previous:   false,
+		Timestamps: true,
+	}
+	logsReq := o.Client.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, logOptions)
+
+	var errs []error
+	if err := o.fileWriter.WriteFromSource(path.Join(destDir, "/gather.logs"), logsReq); err != nil {
```

Comment:
In case multiple must-gather images are run in parallel the individual gather.logs will get interleaved in gather.logs. It probably does not make sense to include them in gather.logs. Skipping all `[must-gather-XXXXX] POD` log lines from `must-gather.logs`.